docs/sources/flow/reference/cli/run.md

```diff
@@ -52,3 +55,17 @@ All components managed by the component controller are reevaluated after
 reloading.
 
 [component controller]: {{< relref "../../concepts/component_controller.md" >}}
+
+## Clustered mode
+
+When the `--cluster.enabled` command-line argument is provided, Grafana Agent will
+start in _clustered mode_.
+
+The agent tries to connect over HTTP/2 to one or more peers provided in the
+comma-separated `--cluster.join-address` list to join an existing cluster,
+otherwise it falls back to bootstrapping a new cluster on its own.
+
+The agent will advertise its address as `--cluster.advertise-address` to other
+nodes; if this is empty it will attempt to find a suitable address to advertise
```

Comment:
I don't follow the advertise-address. What is expected to run at this address? Can you give me an example?

Reply:
Discussed offline - the docs make sense, I somehow didn't connect the dots. Got only one suggestion that could maybe help here - we could say that the agent should be reachable through `--cluster.advertise-address` as it will be advertised to other agents.